docs/misc/deprecated.md

```diff
@@ -88,6 +88,8 @@ are deprecated and replaced with double-dash options (`--opt`):
 The following double-dash options are deprecated and have no replacement:
 
     docker run --networking
+    docker ps --before
+    docker ps --since
```

Comment:
oh, apologies; they're now below "deprecated in 1.8". Can add this to the top of this file (line 15), and use something like:

```
### Docker ps 'before' and 'since' options

**Deprecated In Release: [v1.10.0](https://github.com/docker/docker/releases/tag/v1.10.0)**
 
**Target For Removal In Release: v1.12**
 
The `docker ps --before` and `docker ps --since` options are deprecated in favor
of `docker ps --filter before=...` and `docker ps --filter since=...`.
```